src/pages/clients/common/hooks/useClientColumns.tsx

```diff
@@ -326,11 +325,12 @@ export function useClientColumns() {
           tooltipElement={
             <div className="w-full max-h-48 overflow-auto whitespace-normal break-all">
               <article
-                className="prose prose-sm"
+                className={classNames('prose prose-sm', {
+                  'prose-invert': !reactSettings?.dark_mode,
```

Comment:
@beganovich Color scheme cannot cover all cases here. When text is bolded, the best visibility is when the `prose-invert` class is used for example. But, we need to use reversed logic for tooltips, so on the light mode of the app tooltip is dark so the prose-invert class should be used, however on the dark mode of the app, tooltips are light so it should not be used. Only if we show this outside of the tooltip should we use the prose-invert class when the dark mode of the app is turned on.